### ISSUE TITLE
AC Loads in Mobile view shows only L1 information in a 3-phase system

### ISSUE THREAD

**Issue by dfch** [kwindrem/GuiMods#173]
There is a discrepancy in GuidMods for a three phase setup regarding "AC Loads" information shown between the "Mobile" view and "Simple Overview" and "Tiles" views.

**Current behaviour**

In "Mobile" overview, only the first phase *L1* of all three phases *L1, L2, L3* is shown:

![image](https://github.com/kwindrem/GuiMods/assets/8310360/cdc57c84-f595-41c4-a607-76fc0fa050a6)

![image](https://github.com/kwindrem/GuiMods/assets/8310360/af81e3ac-b34a-4f8e-90c5-2aa0b5986295)

**Comparison to other views**

In "Simple Overview" and "Tiles" views I can see information about all three phases *L1, L2, L3":

![image](https://github.com/kwindrem/GuiMods/assets/8310360/d3c05c26-519e-4ab1-ab82-c741152d0404)

![image](https://github.com/kwindrem/GuiMods/assets/8310360/27ad7aaf-21a7-4495-a4d2-44104fd4fd9e)

**Expected behaviour**

I would expect the complete information of all three phases shown in the "Mobile" view (as shown in the "Simple Overview").

![image](https://github.com/kwindrem/GuiMods/assets/8310360/9ba23f33-7ed6-430a-8587-7d139afaa039)

**Version running**

I am running GuiMods v7.75 on Venus OS v3.01 (on a Victron Cerbo GX).

**Comment by kwindrem** [comment]
A limitation, more than a discrepancy. Obviously there is no space for three phase info in the Mobile Overview AC tiles. The same is true for the stock Mobile Overview. In fact, the stock version only shows total power.

I could probably reduce the font sizes and make room for a second phase but to show three the fonts would be very tiny.

**Comment by dfch** [comment]
I agree. But, instead of showing only 2 of the 3 phases, I would then rather show only Ampère for each phase plus the frequency/Hertz (but leave out voltage and individual Watts per phase). But just showing L1 is a little bit irritating.
But pls do not get me wrong. I can totally live with the current implementation.
